extensions/amp-form/0.1/form-verifiers.js

```diff
@@ -247,7 +248,9 @@ export class AsyncVerifier extends FormVerifier {
       errors.every((error) => previousError.name !== error.name);
     const fixedElements = previousErrors
       .filter(isFixed)
-      .map((e) => this.form_./*OK*/ querySelector(`[name="${e.name}"]`));
+      .map(
+        (e) => formElementsQuerySelectorAll(this.form_, `[name="${e.name}"]`)[0]
```

Comment:
nit: maybe extract `formElementsQuerySelector`. This case may not be problematic since the selector is likely unique, but I'd prefer if we avoid creating and slicing arrays for a single item.


```js
function formElementsQuerySelector(form, query) {
  for (let i = 0; i < form.elements.length; i++) {
    const element = form.elements[i];
    if (matches(element, query)) {
      return element;
    }
  }
  return null;
}
```

```suggestion
        (e) => formElementsQuerySelector(this.form_, `[name="${e.name}"]`)
```